dbms/src/Server/Server.cpp

```diff
@@ -487,6 +490,7 @@ int Server::main(const std::vector<std::string> & args)
 			}
 
 			/// HTTPS
+#if Poco_NetSSL_FOUND
```

Comment:
Same issue here. Need to throw exception if https_port was specified, but library is not available.